### DISCUSSION TITLE
What is the difference between Molecule Parameters "Intestinal Permeability (transcellular)" and "Specific Intestinal Permeability (transcellular)"?

### DISCUSSION THREAD

**Discussion by SteTindal** [Open-Systems-Pharmacology/Forum#1766] [Questions & Problems] [answered]
I'm new to PBPK modelling but I asked a more experienced colleague who uses another software and he did not know the answer either.  I could not find the answer in the documentation.

I have both in an oral PBPK model but it's not clear how each is contributing to the model and they don't have the same value after parameter identification.

**Comment by Proen1** [comment, marked as answer]
I am not entirely sure but I have the ideia  that Specific Intestinal permeability is the one you input the value yourself and the Intestinal Permeability (transcellular) is the one the suite uses for calculation. If, when you make your simulation, you select to use your input value, it takes the Specific Intestinal permeability; if you do not select it, it takes the Calculated Specific intestinal clearance from the internal QSAR that considers MWeff and lipophilicity.
